CCC-Travel-Expense-Policy.md

```diff
@@ -0,0 +1,25 @@
+## Confidential Computing Consortium Travel Expense Policy
+
+
+The Confidential Computing Consortium’s Travel Budgets are intended to enable CCC’s project developers and community members to attend events that they would otherwise be unable to attend due to lack of funding. If you work for a company that has the ability to fund your travel, we ask that you seek funding from your company first, to save funds for those who need it. The goal is to try to use the CCC funds to assist as many people as possible.
+
+In order to maximize spend, travel fund assistance may only be used for:
+
+* Coach airfare tickets 
+* Accommodations (Dates of the event plus one night prior are eligible for reimbursement)
+* Ground transportation to/from the airport
+
+**NOTE:** Funds may not be used for miscellaneous travel expenses including food, visa costs, non-airport transportation, baggage fees, etc. Generally, registration can be reimbursed if the attendee is speaking on behalf of the CCC or staffing a CCC booth at a sponsored event.
+
+
+#### TAC and Outreach Committee Reimbursement Process
+
+1. Estimate travel expenses based on the guidelines listed above.
+2. Submit travel estimate to your committee and operations@confidentialcomputing.io with the reasons for travel (presenting, staffing a booth, etc)
+3. Get approval for the travel from the committee and/or from the Executive Director or the Technical Community Architect
+4. Purchase the travel and attend the event, provide a trip report and/or pictures and video of the event for use on social media. Recap the event in a trip report shared with your committee.
+5. Once you have completed the travel and paid the expenses, fill out the Linux Foundation expense report. Contact operations@confidentialcomputing.io for the expense report template and for any questions. 
```

Comment:
Why the extra step to contact operations to get the template rather than linking to it directly?  Is the template confidential?